### DISCUSSION TITLE
打工的提醒功能不起作用

### DISCUSSION THREAD

**Discussion by LHChina12** [WatsonYao/nian_app#88] [Q&A] [answered]
做测试的时候有反应，但是实际上，过了提醒时间，也没有提示，所以想问问
提醒功能是怎么起作用的。
![Screenshot_20240927_181921](https://github.com/user-attachments/assets/9f322f70-d0e6-42c2-9216-eb497b59b388)
![Screenshot_20240927_190443](https://github.com/user-attachments/assets/b34f3f13-ef06-49cf-88ba-a87c3c516262)

**Comment by WatsonYao** [comment, marked as answer]
帮助文档里面的QA和最下面那个权限相关 都留意下。
